### DISCUSSION TITLE
What's the best way to prevent search engines from indexing my Photoprism instance?

### DISCUSSION THREAD

**Discussion by 0xDedSec42** [photoprism/photoprism#4571] [Q&A] [answered]
As mentioned in the subject, I want to prevent search engines from indexing my Photoprism instance. How can I achieve this? The most obvious way would be to add a robot.txt but how and where?

**Comment by lastzero** [comment, marked as answer]
With these changes, all search engine robots should be blocked (at least until there is a [public image wall](https://github.com/photoprism/photoprism/issues/669)):

- https://github.com/photoprism/photoprism/issues/4574

You will be able to test this with our preview build shortly:

- https://docs.photoprism.app/getting-started/updates/#development-preview

If you use a reverse proxy, you can alternatively set the `X-Robots-Tag` header there (or serve a custom `robots.txt` file):

- https://docs.photoprism.app/getting-started/proxies/traefik/

**Reply by 0xDedSec42** [reply]
Wow, that was fast. Thank you very much.
